Formula/rpg-cli.rb

```diff
@@ -0,0 +1,17 @@
+class RpgCli < Formula
+  desc "Your filesystem as a dungeon!"
+  homepage "https://github.com/facundoolano/rpg-cli"
+  url "https://github.com/facundoolano/rpg-cli/archive/refs/tags/0.4.1.tar.gz"
+  sha256 "2b68e9c53ff7fcebb319aa741e008b1f8c2ed714558302c1707a1fc87fa09164"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_equal "rpg-cli #{version}", shell_output("#{bin}/rpg-cli --version").strip
```

Comment:
Can you propose a better test here than checking the version output? Something that tests the functionality of the program is better, as `version` and `help` output comparisons usually pass even for broken installs.

Reply:
Hi Nanda! Thanks for the feedback! Let mw know if the new version needs more polishing :)

Reply:
Seems okay to me 👍🏼.